suite-native/module-onboarding/src/components/AssetsOverview.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react';
+
+import { CryptoIcon } from '@trezor/icons';
+import {
+    Box,
+    Card,
+    Input,
+    InputWrapper,
+    SelectableListItem,
+    Text,
+    VStack,
+} from '@suite-native/atoms';
+import { AccountInfo } from '@trezor/connect';
+
+type AssetsOverviewProps = {
+    accountInfo: AccountInfo | null;
+};
+
+export const AssetsOverview = ({ accountInfo }: AssetsOverviewProps) => {
+    const [selectedDevice, setSelectedDevice] = useState<string>('firstSelectable');
+    const [assetName, setAssetName] = useState<string>('');
+
+    const handleSelectDevice = (value: string | number) => {
+        setSelectedDevice(value.toString());
+    };
+
+    return (
+        <Card>
+            <Box marginTop="large" marginBottom="medium">
+                <Box alignItems="center" justifyContent="center" marginBottom="medium">
+                    <CryptoIcon name="btc" size="large" />
+                    <Box marginTop="large" marginBottom="small">
+                        <Text variant="titleSmall" color="black">
+                            {accountInfo?.balance}
+                        </Text>
+                    </Box>
+                    <Text variant="label" color="black">
+                        ≈ 0.0003333 BTC
```

Comment:
is this right? shouldn't be there some value from accountInfo?

Reply:
there is follow-up task for this https://github.com/trezor/trezor-suite/issues/5927